examples/dt/Adoption/DFG-upgrades-adoption-18.0-from-17.1-passed-phase2-rhev-3cont-2comp-ipv4-HA-geneve-ovn/DFG-upgrades-adoption-18.0-from-17.1-passed-phase2-rhev-3cont-2comp-ipv4-HA-geneve-ovn.md

```diff
@@ -0,0 +1,54 @@
+# Deployed Topology DFG-upgrades-adoption-18.0-from-17.1-passed-phase2-rhev-3cont-2comp-ipv4-HA-geneve-ovn
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | 22.12.23      |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Openshift master/worker combo-node cluster    | vm    | 3      |
+| Controller nodes - Source osp 17              | vm    | 3      |
```

Comment:
Is there an undercloud node as well? I recall at some point there was but i'm not sure if that's still the case or we're using some other node as a "temporary undercloud" now.

Reply:
Added UC node as it is being used in deployment of osp.